src/Core/Test/Stub/Framework/IdsCollection.php

```diff
@@ -142,7 +142,7 @@ public function getKeys(array $ids): string
             if ($key) {
                 $keys[] = $key;
             } else {
-                throw new \RuntimeException('Key not found for id ' . $id);
+                $keys[] = 'Key not found for ID ' . $id;
```

Comment:
this method is only used in messages of assert methods, so it is imo not very useful if it throws exceptions, as it covers the real failing assertion